plugins/permission-backend/src/service/router.ts

```diff
@@ -141,9 +139,11 @@ const handleRequest = async (
       identity: {
         type: 'user',
         userEntityRef: credentials.principal.userEntityRef,
-        ownershipEntityRefs,
+        ownershipEntityRefs: info.ownershipEntityRefs,
       },
       token,
+      credentials,
+      info,
```

Comment:
Where the `expiresInSeconds` is defined?

Reply:
It's in the credentials, although not particularly important 😅